tests/algorithms/test_cutmix.py

```diff
@@ -61,7 +62,8 @@ def test_cutmix(self, fake_data, alpha):
                                             H=x_fake.shape[3],
                                             cutmix_lambda=cutmix_lambda,
                                             cx=cx,
-                                            cy=cy)
+                                            cy=cy,
+                                            uniform_sampling=uniform_sampling)
         bbox = (bbx1, bby1, bbx2, bby2)
         # Adjust lambda
         cutmix_lambda = 1 - ((bbx2 - bbx1) * (bby2 - bby1) / (x_fake.size()[-1] * x_fake.size()[-2]))
```

Comment:
does uniform sampling need to be passed to cutmix_batch below too?

Reply:
Technically no, because if the explicit bounding box is given then the sampling step is skipped. I agree it looks odd though. Will fix.